test/functional/plugin/lsp/utils_spec.lua

```diff
@@ -301,4 +301,36 @@ describe('vim.lsp.util', function()
       end)
     end)
   end)
+
+  describe('open_floating_preview', function()
```

Comment:
Why are the tests not removed from the old position?

Reply:
because before inside the `make_floating_popup_options` function describe block .

Reply:
So why aren't they removed?